.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,10 +1,17 @@
 ---
-name: Bug report
-about: Create a report to help us improve
-
+name: "\U0001F41B Bug Report"
+about: Create a bug report to help us improve dvc
+labels: bug, triage
 ---
 
+## Bug Report
+
```

Comment:
```suggestion

Please provide an explanation of the problem and steps to reproduce.

```

Reply:
I'd just leave `steps to reproduce` for now. I expect it, but, I'll leave it to the *reporters*.

Reply:
Sounds good.